app/src/UI/LegacyMap/helpers/components/DrawControls.tsx

```diff
@@ -241,18 +259,32 @@ const DrawControls = () => {
             'line-cap': 'round',
             'line-join': 'round'
           },
-          filter: ['all', ['==', 'active', 'false']],
+          filter: ['all', ['==', 'active', 'false'], ['!=', 'user_error', 'true']],
           paint: {
             'line-color': '#FCBA19',
             'line-width': 3
           }
         },
+        {
+          id: 'gl-error-line',
+          type: 'line',
+          layout: {
+            'line-cap': 'round',
+            'line-join': 'round'
+          },
+          filter: ['all', ['==', 'active', 'false']],
+          paint: {
+            'line-color': ['match', ['get', 'user_error'], 'true', 'red', 'false', '#FCBA19', '#FCBA19'],
```

Comment:
```suggestion
            'line-color': ['match', ['get', 'user_error'], 'true', '#B00020', 'false', '#FCBA19', '#FCBA19'],
```

I just recommend something a little just jarring than the web-safe red. This is the hex used by MUI for errors.

Req.
![image](https://github.com/user-attachments/assets/c701aeff-b21a-45b7-8aef-aa8814e7d5cc)

Curr.
![image](https://github.com/user-attachments/assets/a3e456fc-d011-4fd5-956c-436d826973df)